_posts/2018-12-24-ApiaryUI.md

```diff
@@ -0,0 +1,131 @@
+---
+title: ApiaryUI - Interactive Documentation v4
+excerpt: Introduction version 4 of Apiary Interface Documentation
+layout: post
+date: 2018-12-24 12:00:00 +0200
+author: char0n
+published: true
+comments: true
+---
+
+After months of hard work and burning the midnight oil I am very proud to announce
+the **ApiaryUI** - brand new, fully reviewed and refactored version of 
```

Comment:
I think “fully reviewed” is a little awkward in this sentence, although I understand your intent.

Perhaps re-imagined, or rebuilt from the ground up could work better.

Reply:
used: `rethought`

Reply:
Agreed. I think in the fist paragraph, we should focus on what does it bring for user, and in that sense, first thing on my mind is speed.